StarWars5e.Parser/Parsers/MonsterManualProcessor.cs

```diff
@@ -21,9 +21,15 @@ public override Task<List<Monster>> FindBlocks(List<string> lines)
 
             for (var i = 0; i < lines.Count; i++)
             {
-                if(lines[i].StartsWith("## ") || lines[i].StartsWith("### "))
+                var canBeSectionText = lines[i].StartsWith("## ");
+                var canBeFlavorText = lines[i].StartsWith("### ");
+
+                if (canBeSectionText || canBeFlavorText)
                 {
-                    var flavorTextEndIndex = lines.FindIndex(i, f => f == "___" || f.StartsWith("### "));
+                    var flavorTextEndIndex = canBeSectionText 
+                        ? lines.FindIndex(i, f => f == "___" || f.StartsWith("### ")) 
+                        : lines.FindIndex(i, f => f == "___");
```

Comment:
was incorrectly skipping over all entries with "### " (flavor text) because it would always match on itself in that case, instead of looking for the "____" ending.